src/components/shared-player-components/volume/volume-panel.tsx

```diff
@@ -13,11 +13,17 @@ export const VolumePanel: VoidComponent = () => {
 
   const onVolumeInputHandler = (e: InputEvent) => {
     const inputEl = e.target as HTMLInputElement
-    const parsedValue = parseInt(inputEl.value, 10)
+    const parsedValue = Math.pow(1, 2)
```

Comment:
You need to take value from `input.value` right now slider does nothing.

Reply:
Ah I did not intend to change that. thanks!